Redirect power module reading to specific device power monitor PVs

gustavosr8 opened the pull request:
Since the device itself can't read the power consumption, this information has to come from the power module specific channel that is related to each device. The power module used is PM2 because PM4 is just used for redundancy and to replace the PM2 in case of some failure, and don't have any device connected to it's channels.

ericonr commented on ipmiMgrApp/Db/power_mngr.template:
```diff
@@ -0,0 +1,6 @@
+record(ai, "$(P)$(R)$(id)Pwr") {
+  field(DESC, "Steady state power draw")
+  field(SCAN, "10 second")
+  field(INP, "$(P)$(R)PM2CurrCh$(power_channel)-Mon")
+  field(EGU, "W")
+}
```
Why couldn't this use an alias?

Also, if you want, instead of having a scan, you can simply omit the `SCAN` field and use a `{your_pv} CP` link and it will be processed everytime the power draw changes.

henriquesimoes reviewed (commented):
I think this should be built on top of #5, if we can wait for it to be merged.

ericonr commented on ipmiMgrApp/Db/afcv3.substitutions:
```diff
@@ -27,6 +27,12 @@
 #  19   Compact  0xf0    0xc1  0x69  0x10              HS 013 AMC9
 # ------------------------------------------------------------------
 
+file "power_mngr.db"
+{
+    pattern { id    , power_channel }
+            { $(id) , $(power_channel) }
+}
```
For a template like this, you can simply do

```
file "power_mngr.db"
{
    {}
}
```

The existing substitutions are already forwarded to templates automatically, so you don't need to specify them. All you need to do is instantiate that file once.

If you prefer to do it like this for documentation purposes though, feel free.

gustavosr8 committed:
Redirect power module reading to specific device power monitor PVs

Since the device itself can't read the power consumption, this
information has to come from the power module specific channel that is
related to each device. The power module used is PM2 because PM4 is just
used for redundancy and to replace the PM2 in case of some failure, and
doesn't have any device connected to its channels.

henriquesimoes commented on ipmiMgrApp/Db/Makefile:
```diff
@@ -16,6 +16,7 @@ DB += sensor_ai_alias.db
 DB += fru_common_alias.db
 DB += fru_cu_common_alias.db
 DB += system_common_alias.db
+DB += power_mngr.db
```
Should this be `power_module.db`? I could not find a `power_mngr` template in [slac-epics-modules/ipmiComm](https://github.com/slac-epics-modules/ipmiComm) nor a substitution with this name.

henriquesimoes reviewed (approved):
Just some typos in the commit message:

> The power module used is PM2 because PM4 is just used for redundancy and to replace the PM2 in case of some failure, and **doesn't** have any device connected to **its** channels.
